tests/components/alarm_control_panel/test_manual.py

```diff
@@ -27,6 +29,13 @@ def tearDown(self):  # pylint: disable=invalid-name
         """Stop down everything that was started."""
         self.hass.stop()
 
+    def test_setup_demo_platform(self):
+        """Test setup."""
+        mock = MagicMock()
+        add_devices = mock.MagicMock()
```

Comment:
What's this?

Reply:
it's the code you originally suggested.  I had to import MagicMock as it wasn't already imported in this file and and to define what ```mock``` was.

Reply:
I think it should be like this, since you import `MagicMock`:
```py
add_devices = MagicMock()
```